packages/calcite-components/src/custom-theme.stories.ts

```diff
@@ -104,7 +105,9 @@ const kitchenSink = (args: Record<string, string>, useTestValues = false) =>
           <div>${checkbox}</div>
           ${chips} ${pagination} ${slider}
         </div>
-        <div class="demo-column">${datePicker} ${tabs} ${loader} ${calciteSwitch}</div>
+        <div class="demo-column">${datePicker} ${tabs} ${loader} ${calciteSwitch} 
+        <div style="margin-top: 40px">${popover}</div>
```

Comment:
The comment from https://github.com/Esri/calcite-design-system/pull/10267/files#r1753091966 applies here.